### PR TITLE
update cose typ language

### PR DESCRIPTION
**Pull request by decentralgabe**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/pull/318.html" title="Last updated on Oct 24, 2024, 4:00 AM UTC (1f816f4)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/318/1dd153d...1f816f4.html" title="Last updated on Oct 24, 2024, 4:00 AM UTC (1f816f4)">Diff</a>

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

**Comment by decentralgabe** [comment]
@iherman requesting permission to merge this before a week to avoid needing to delay CR2

File: index.html [hunk 1]
```diff
@@ -846,14 +854,12 @@ <h2 id="securing-vcs-with-cose">Securing JSON-LD
...
+            The <code>typ (16)</code> header parameter, as described in
             <a data-cite="RFC9596#section-2">COSE "typ" (type) Header Parameter</a>,
             SHOULD be <code>application/vc+cose</code>.
-            When present, the <code>content type (3)</code> header parameter
-            SHOULD be <code>application/vc</code>.
-            The <code>content type (3)</code> header parameter value can be used
-            to differentiate between secured content of different types when using
-            <code>application/vc+cose</code>.
+            The <code>content type (3)</code> header parameter SHOULD be <code>application/vc</code>.
+            The <code>content type</code> header parameter is optional, and can be used
```
Comment (selfissued) [inline index.html:861]:
```suggestion
            The <code>content type (3)</code> header parameter is optional, and can be used
```

File: index.html [hunk 2]
```diff
@@ -918,10 +924,15 @@ <h2 id="securing-vps-with-cose">Securing JSON-LD Verifiable Presentations with C
...
-            between secured content of different types when using <code>application/vp+cose</code>.
+            The <code>typ (16)</code> header parameter, as described in
+            <a data-cite="RFC9596#section-2">COSE "typ" (type) Header Parameter</a>,
+            SHOULD be <code>application/vp+cose</code>.
+            The <code>content type (3)</code> header parameter SHOULD be <code>application/vp</code>.
+            The <code>content type</code> header parameter value can be used
+            to differentiate between secured content of different types when using
+            <code>application/vp+cose</code>.
+            The <code>content type</code> header parameter is optional, and can be used
+            to express a more specific media type than <code>application/vc</code> when one is available.
```
Comment (selfissued) [inline index.html:935]:
```suggestion
            The <code>content type (3)</code> header parameter is optional, and can be used
            to express a more specific media type than <code>application/vp</code> when one is available.
```

**Review by selfissued — APPROVED** [review]

**Comment by iherman** [comment]
> @iherman requesting permission to merge this before a week to avoid needing to delay CR2

Yeah, it is o.k.. The important point is to refresh https://w3c.github.io/vc-jose-cose/transitions/CR2/2024-11-05/ . That is the reference on the submission request.

**Comment by decentralgabe** [comment]
merging and will refresh the CR link

**Pull request merged**